### PR TITLE
Clarify oracle URL property behavior

### PR DESCRIPTION
**Pull request by KyleAure**
Update metatype to clarify the expected behavior that configuring a URL should override the individual properties of `driverType`, `serverName`, `databaseName`, and `portNumber` based on the documented behavior of the Oracle JDBC driver.  See oracle's documentation here: https://javadoc.io/static/com.oracle.database.jdbc/ojdbc8/21.6.0.0.1/oracle/jdbc/datasource/OracleCommonDataSource.html#setURL-java.lang.String-

### CODE REVIEW

**Comment by youngjane99** [comment]
Hello Kyle, the change looks good, my question is: will the change to be synchronized into Liberty information center which client and L2 normally reference when checking the properties, for example:
https://www.ibm.com/docs/en/was-liberty/base?topic=configuration-openidconnectprovider#oauthProvider/databaseStore/dataSource/properties.oracle
and other liberty information center URLs that contain datasource's properties?

Thanks for your time to make it clear.
Yang Jie

**Review by frowe — APPROVED** [review]

**Comment by KyleAure** [comment]
@youngjane99 
> will the change to be synchronized into Liberty information center which client and L2 normally reference when checking the properties

Yes, all this doc is auto-generated from our metatype and is updated with every long-term release of Liberty.

**Comment by KyleAure** [comment]
#build

**Comment by posmith** [comment]
L2 message review not required

**Comment by KyleAure** [comment]
Just waiting on a review from ID.  FYI @OpenLiberty/message-reviewer

**Review by hiltoncv — COMMENTED** [review]
Consider this text instead of the original text:
"URL.oracle.desc=JDBC driver property: URL. The URL for connecting to the database. If the URL is configured and it contains values for properties such as serverName and driverType, the values that are specified in the URL override the values that are specified for the individual properties. Examples: jdbc:oracle:thin:@//localhost:1521/sample or jdbc:oracle:oci:@//localhost:1521/sample."

**Comment by frowe** [comment]
The proposed revision needs to include the word "could" because we don't control (or necessarily know from release to release) what the exact behavior of the driver will be. "the values that are specified in the URL **could** override the values "

File: dev/com.ibm.ws.jdbc.metatype/resources/OSGI-INF/l10n/metatype.properties
Comment (hiltoncv) [inline dev/com.ibm.ws.jdbc.metatype/resources/OSGI-INF/l10n/metatype.properties]: Proposed text:
"JDBC driver property: URL. URL for connecting to the database. If a URL is configured that contains values for other properties like serverName and driverType, the Oracle JDBC driver overrides the values specified for those individual properties with the values specified in the URL. Oracle JDBC driver updates might impact this behavior. "

Comment (KyleAure) [reply dev/com.ibm.ws.jdbc.metatype/resources/OSGI-INF/l10n/metatype.properties]: Updated description based on feedback and offline discussions

**Comment by KyleAure** [comment]
#build

**Comment by KyleAure** [comment]
Build failure unrelated to change.

**Comment by KyleAure** [comment]
#libby

**Pull request merged**
